python/tests/global/test_formatting.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import os
+
+import pytest
+
+from .excluded_files import get_excluded_files
+
+
+def get_py_files(path):
+    fnames = []
+    for root, dirs, files in os.walk(path):
+        for file in files:
+            if file.endswith(".py"):
+                fnames.append(os.path.join(root, file))
+    return fnames
+
+
+@pytest.mark.skipif(sys.version_info < (3, 6), reason="requires Python3")
+def test_code_style():
+    from click.testing import CliRunner
+    import black
+    from pathlib import Path
+
+    root = str(Path(__file__).parent.parent.parent)
+
+    excluded_files = get_excluded_files(root)
+    all_py_files = get_py_files(root)
+    files_to_test = set(all_py_files) - set(excluded_files)
+
+    runner = CliRunner()
+
+    resp = runner.invoke(black.main, ["--check"] + list(files_to_test))
+    assert (
```

Comment:
Maybe just import `unittest`, make a testcase and use `self.assertEqual(resp.exit_code, 0, "msg…")`?

Reply:
Because this test is already depending on `pytest`, especially with the `@pytest.mark.skipif(sys.version_info < (3, 6), reason="requires Python3")` above it, I feel it is more consistent to use `assert`, which is the `pytest` way, instead of `unittest`.